_includes/story.md

```diff
@@ -1118,29 +1137,33 @@ end
 
 ### Workerholic compared to the Gold Standard: Sidekiq
 
-Finally, we wanted to compare with Sidekiq one last time with each types of jobs individually. We're on par with Sidekiq, and only slightly faster than Sidekiq each time, and as we mentioned before this is because Sidekiq is a more mature and robust solution with a much larger feature-set.
+Finally, we wanted to compare Workerholic with Sidekiq one last time with different types of jobs individually. We're on par with Sidekiq, and only slightly faster than Sidekiq each time, and as we mentioned before this is because Sidekiq is a more mature and robust solution with a much larger feature-set.
 
+{: .center}
 ![benchmark_workerholic_sidekiq](/img/benchmark_workerholic_sidekiq.png)
 
-We also decided to compare the results of JRuby vs MRI. Because jRuby can run in parallel without the need of spinning up multiple processes, we found that execution of CPU blocking jobs was much faster in JRuby than in MRI, which is what we would expect.
+As an additional feature, we wanted to make sure Workerholic runs on various Ruby interpeters and that's why we chose JRuby (one of the most stable and mature Ruby interpreters) to be the main alternative interpreter. To ensure full compatibility, we ran an extensive set of jobs on our Rails application.
+We also decided to benchmark JRuby against MRI. Because JRuby can run in parallel without the need of spinning up multiple processes, we found that execution of CPU blocking jobs was much faster in JRuby than in MRI, which is what we would expect.
```

Comment:
either add a `<br>` if you want 2 different paragrpahs without a space between them. Or add another new line to have 2 paragraphs spaced out. Right now, just a carriage return will not do anything, the second paragraph will just be part of the first.